index.html

```diff
@@ -724,9 +730,35 @@ <h2>Security Considerations</h2>
 critical systems using the technology outlined in this specification.
     </p>
 
-    <p class="issue">
-Write security considerations.
-    </p>
+    <section class="informative">
+      <h3>Bitstring Encoding</h3>
+
+      <p>
+It is critical that implementers pay particular attention to the way that they
+encode and decode bitstrings. Failure to do so can result in a failure to check
+the correct position in a bitstring. Checking an incorrect bitstring index can
+lead to failures such as mistaking a revoked credential for one that is not
+revoked. As stated in Section <a href="#statuslist2021credential"></a>,
+bitstrings are encoded as big-endian values with the index starting at the
+left-most index of the bitstring array. The diagram below demonstrates the
+proper layout for an uncompressed bitstring.
```

Comment:
```suggestion
It is critical that implementers pay particular attention to the way that they
encode and decode bitstrings. Failure to do so can result in checking the
wrong bitstring index for a given credential, leading to a misinterpretation
of its present state (e.g., mistakening a revoked status for an unrevoked
status). As stated in Section <a href="#statuslist2021credential"></a>,
bitstrings are encoded such that the first (zeroth) index refers to the
left-most bit of the bitstring array. The diagram below demonstrates the
proper layout for an uncompressed bitstring.
```

Reply:
`s/mistakening/mistaking/` and I'm good with this change.

Reply:
Fixed.